From 3129c56aea05dd2b7e88c76bb2050fded0529243 Mon Sep 17 00:00:00 2001
From: Aurash Karimi <a.karimi@imperial.ac.uk>
Date: Wed, 19 Nov 2025 09:48:11 +0000
Subject: [PATCH] remove unrecognised parameter

---
 book.toml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/book.toml b/book.toml
index 41fba09c..84d181ee 100644
--- a/book.toml
+++ b/book.toml
@@ -1,7 +1,6 @@
 [book]
 authors = ["Alex Dewar"]
 language = "en"
-multilingual = false
 src = "docs"
 title = "MUSE2"

--
2.53.0
